contrib/profiler/Profiler.h

```diff
@@ -11,7 +11,7 @@
 #undef fastcall
 
 //#define USE_CHRONO
-#if !defined(USE_CHRONO) && defined(__arm__)
+#if !defined(USE_CHRONO) && defined(__arm__) || defined(__aarch64__)
```

Comment:
> that won't compile

Test case for what you proposed:
```cpp
#include <stdio.h>

#if !defined(USE_CHRONO) && defined(__arm__) || defined(__aarch64__)
#define USE_CHRONO
#endif

int main ()
{
	printf("USE_CHRONO: %s\n",
#ifdef USE_CHRONO
	"defined"
#else
	"not defined"
#endif
	);

	return 0;
}
```
```
% cc test.c && ./a.out
USE_CHRONO: not defined

% cc -DUSE_CHRONO test.c && ./a.out
USE_CHRONO: defined

% cc -D__arm__ test.c && ./a.out
USE_CHRONO: defined

% cc -D__aarch64__ test.c && ./a.out
USE_CHRONO: defined

% cc -DUSE_CHRONO -D__arm__ test.c && ./a.out
USE_CHRONO: defined

% cc -DUSE_CHRONO -D__aarch64__ test.c && ./a.out
test.c:4:9: warning: 'USE_CHRONO' macro redefined
#define USE_CHRONO
        ^
<command line>:1:9: note: previous definition is here
#define USE_CHRONO 1
        ^
1 warning generated.
USE_CHRONO: defined
```

Test case for what I proposed:
```cpp
#include <stdio.h>

#if !defined(USE_CHRONO) && (defined(__arm__) || defined(__aarch64__))
#define USE_CHRONO
#endif

int main ()
{
	printf("USE_CHRONO: %s\n",
#ifdef USE_CHRONO
	"defined"
#else
	"not defined"
#endif
	);

	return 0;
}
```
```
% cc test2.c && ./a.out
USE_CHRONO: not defined

% cc -DUSE_CHRONO test2.c && ./a.out
USE_CHRONO: defined

% cc -D__arm__ test2.c && ./a.out
USE_CHRONO: defined

% cc -D__aarch64__ test2.c && ./a.out
USE_CHRONO: defined

% cc -DUSE_CHRONO -D__arm__ test2.c && ./a.out
USE_CHRONO: defined

% cc -DUSE_CHRONO -D__aarch64__ test2.c && ./a.out
USE_CHRONO: defined
```
Did you see where your part is not correct?